.github/workflows/test.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
```

Comment:
Django 3.2 still supports python 3.6 and 3.7, we should strive to maintain the same range of compatibility.

Reply:
I agree on that we ideally also should run the 3.2 tests with Python 3.6 and 3.7. The problem is that Django 4 doesn't support those versions. I'll see if I can come up with some simple solution for including those Python versions, but only running the tests using compatible versions.

Reply:
Seems this was easy to do 🙂

Reply:
I forgot python 3.6 was EOL in Dec too. so we probably don't need to support it.. but we should keep 3.7 in our list.